html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <div><input type="email" name="email" /></div>
+      <div>Why is coding cool?</div>
```

Comment:
This should be a `label` element

Reply:
Are you referring to the `div` element on line 11?